packages/eslint-plugin-template/tests/rules/prefer-ngsrc/cases.ts

```diff
@@ -0,0 +1,86 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import type { MessageIds } from '../../../src/rules/prefer-ngsrc';
+
+const missingAttribute: MessageIds = 'missingAttribute';
+const invalidDoubleSource: MessageIds = 'invalidDoubleSource';
+
+export const valid = [
+  '<img ngSrc="http://localhost">',
+  "<img [ngSrc]=\"'http://localhost'>",
+  '<img [ngSrc]="value">',
+];
+
+export const invalid = [
+  convertAnnotatedSourceToFailureCase({
+    description: 'should fail image when using src over ngsrc',
+    annotatedSource: `
+      <ng-template>
+        <img src="http://localhost">
+             ~~~~~~~~~~~~~~~~~~~~~~
+        <img [src]="'http://localhost'">
+             ^^^^^^^^^^^^^^^^^^^^^^^^^^
+        <img [src]="value">
+             #############
+      </ng-template>
+      `,
+    messages: [
+      {
+        char: '~',
+        messageId: missingAttribute,
+      },
+      {
+        char: '^',
+        messageId: missingAttribute,
+      },
+      {
+        char: '#',
+        messageId: missingAttribute,
+      },
+    ],
+  }),
+  convertAnnotatedSourceToFailureCase({
+    description: 'should fail image when using both src and ngsrc',
+    annotatedSource: `
+      <ng-template>
+        <img ngSrc="http://localhost" src="http://localhost">
```

Comment:
I read from the source that the error will always be at the `src`-attribute, and not necessarily the second attribute. 
Might be a good idea to add a test where `src` and `ngSrc` are reversed.

Reply:
Agreed, thanks @jerone!

Reply:
Fixed.